main.tex

```diff
@@ -222,7 +222,7 @@ \subsection{Astropy development model}
 of code.} contributed by $232$ unique contributors over $19270$ \texttt{git}
 commits.
 \figurename~\ref{fig:ncommits}, left, shows the distribution of total number of
-commits per contributor as of November 2017.
+commits per contributor as of early 2018.
```

Comment:
nitpicking and probably it's not worth the effort of replotting, but I think it would be more consistent to cut these graph at the time of the v2.0 release.

Reply:
I think it's OK to have figures to go the time when they were made.